### PR TITLE
Move document from g/documentation

### PR DESCRIPTION
**Pull request by Kristian-ZH**
**How to categorize this PR?**

/area documentation
/kind enhancement
/platform azure

**What this PR does / why we need it**:
Move document from https://github.com/gardener/documentation/tree/master/website/documentation/guides/administer_shoots/gardener_azure because the documents there are not maintained.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator
None
```

### CODE REVIEW

File: docs/tutorials/kubernetes-cluster-in-azure-with-gardener/_index.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+    You must [request a new service account](https://jira.multicloud.int.sap/plugins/servlet/desk/portal/2) first, if you don't have one.
+
+    <img src="images/request_spn.jpg">
+
+
+
+    Naming Convention for the Service UserID: `azrspn_<PartOfSubName>_usecase`
+
+    <img src="images/request_spn1.jpg">
```
Comment (ialidzhikov) [inline docs/tutorials/kubernetes-cluster-in-azure-with-gardener/_index.md:32]: I think this step should be reworked as it describes SAP specific process.

File: docs/tutorials/kubernetes-cluster-in-azure-with-gardener/_index.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        Secrets for the Azure Account/Service Principal can be genereted/rotated via the Azure Portal.
+        Access the [Azure Portal](https://portal.azure.com) and navigate to the `Active Directory` service.
+        Within the service navigate to `App registrations` and select your service principal. 
+        In the detail view navigate to `Certificates & secrets`. In the section, you can generate a new secret for the Service Principal.
+
+3. On the Gardener dashboard, choose *Secrets* and then the plus sign <img src="images/plus_icon.jpg"> in the Azure frame to add a new Azure secret.
+
+    <img src="images/gardenernewazuresecret.jpg">
+
+5. Provide the details for the Azure service account.
```
Comment (ialidzhikov) [inline docs/tutorials/kubernetes-cluster-in-azure-with-gardener/_index.md:61]: Right now there is not step 4, the tutorial is jumping from step 3. to 5.

```suggestion
4. Provide the details for the Azure service account. 
```

File: docs/tutorials/kubernetes-cluster-in-azure-with-gardener/_index.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+---
```
Comment (ialidzhikov) [inline docs/tutorials/kubernetes-cluster-in-azure-with-gardener/_index.md:1]: I don't see `images/gardenerazuresecret.jpg` being linked/used by this tutorial. Can we remove it or make the tutorial use it?

Comment (ialidzhikov) [inline docs/tutorials/kubernetes-cluster-in-azure-with-gardener/_index.md:1]: I don't see `images/gardenerazuresecret0.jpg` being linked/used by this tutorial. Can we remove it or make the tutorial use it?

File: docs/tutorials/kubernetes-cluster-in-azure-with-gardener/_index.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    Before you can provision and access a Kubernetes cluster on Azure, you need to add the account credentials in Gardener. 
+    Gardener needs the credentials to provision and operate the Azure infrastructure for your Kubernetes cluster.
+
+    **Ensure that the account has the `contributor` role.**
+    ```
+
+
+    -  Tenant ID
+    
+        The TenantID is also called DirectoryID - https://portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/Properties.
```
Comment (dkistner) [inline docs/tutorials/kubernetes-cluster-in-azure-with-gardener/_index.md:32]: I guess the term `DirectoryID` is not a valid one. 
Can we remove the portal link here?

Could you refer here to the official Azure docs: https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-how-to-find-tenant

File: docs/tutorials/kubernetes-cluster-in-azure-with-gardener/_index.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+        <img src="images/azuregettenantid.jpg">
+
+    -  ClientID
+        Select the subscription.
+        <img src="images/azureselectsubscription.jpg">
+
+    - Select the SPN.
+        <img src="images/azureselectspn.jpg">
+
+    *Note:* The ClientID is also called ApplicationID.
```
Comment (dkistner) [inline docs/tutorials/kubernetes-cluster-in-azure-with-gardener/_index.md:42]:
```suggestion
    *Note:* A service principal consist of a `ClientID` (also called `ApplicationID`) and a Client Secret
```

We could also refer here to the official Azure docs: https://docs.microsoft.com/en-us/azure/active-directory/develop/app-objects-and-service-principals

File: docs/tutorials/kubernetes-cluster-in-azure-with-gardener/_index.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+4. Provide the details for the Azure service account.  
+    After processing the ticket, you’ll receive the Service Principle credentials via email.
+    Copy "Key Value" from the email into "Client Secret".
+
+    <img src="images/gardeneraddazuresecret.jpg">
+
+5. To create a new cluster, choose *Clusters* and then the plus sign in the lower right corner.
+
+    <img src="images/new_cluster.jpg">
```
Comment (dkistner) [inline docs/tutorials/kubernetes-cluster-in-azure-with-gardener/_index.md:63]: I guess this picture is outdated. Can we provide new one?

File: docs/tutorials/kubernetes-cluster-in-azure-with-gardener/_index.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+level: beginner
+category: Getting Started
+scope: app-developer
+---
+
+# Create shoot clusters in Azure
+
+### Prerequisites
+
+-   You need an Azure account.
```
Comment (dkistner) [inline docs/tutorials/kubernetes-cluster-in-azure-with-gardener/_index.md:12]: You need to have as well as Azure Service Principal assigned to your subscription or the permissions to create one in the Active Directory tenant and assign it to your subscription.

File: docs/tutorials/kubernetes-cluster-in-azure-with-gardener/_index.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+        Within the service navigate to `App registrations` and select your service principal. 
+        In the detail view navigate to `Certificates & secrets`. In the section, you can generate a new secret for the Service Principal.
+
+3. On the Gardener dashboard, choose *Secrets* and then the plus sign <img src="images/plus_icon.jpg"> in the Azure frame to add a new Azure secret.
+
+    <img src="images/gardenernewazuresecret.jpg">
+
+4. Provide the details for the Azure service account.  
+    After processing the ticket, you’ll receive the Service Principle credentials via email.
+    Copy "Key Value" from the email into "Client Secret".
```
Comment (dkistner) [inline docs/tutorials/kubernetes-cluster-in-azure-with-gardener/_index.md:57]:
```suggestion
4. Enter the `TenantID`, `SubscriptionID` and the Service Principal credentials (`ClientID` and `ClientSecret`) into the Secret definition.
```

File: docs/tutorials/kubernetes-cluster-in-azure-with-gardener/_index.md [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+    *Note:* The ClientID is also called ApplicationID.
+    <img src="images/azuregetclientid.jpg">
+
+    - Client Secret
+        Secrets for the Azure Account/Service Principal can be genereted/rotated via the Azure Portal.
+        Access the [Azure Portal](https://portal.azure.com) and navigate to the `Active Directory` service.
+        Within the service navigate to `App registrations` and select your service principal. 
+        In the detail view navigate to `Certificates & secrets`. In the section, you can generate a new secret for the Service Principal.
+
+3. On the Gardener dashboard, choose *Secrets* and then the plus sign <img src="images/plus_icon.jpg"> in the Azure frame to add a new Azure secret.
```
Comment (dkistner) [inline docs/tutorials/kubernetes-cluster-in-azure-with-gardener/_index.md:51]: I guess the `plus_icon.jpg` is not really required.

File: docs/tutorials/kubernetes-cluster-in-azure-with-gardener/_index.md [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+-   You need an Azure account.
+-   You have access to the Gardener dashboard and have permissions to create projects.
+
+
+1. Go to the Gardener dashboard and create a *Project*.
+
+    <img src="images/new_gardener_project.jpg">
+
+
+2. Get properties of your Azure account/Service Principal.
```
Comment (dkistner) [inline docs/tutorials/kubernetes-cluster-in-azure-with-gardener/_index.md:21]:
```suggestion
2. Get properties of your Azure AD tenant, Subscription and Service Principal.
```

**Comment by dkistner** [comment]
As described here: https://github.com/gardener/documentation/issues/287
We will update the guides (updated figures/screenshots, more references to cloud provider documentations etc.) in a subsequent step.
So we go on here.
/lgtm

**Pull request merged**